resources/views/partials/content.blade.php

```diff
@@ -3,7 +3,7 @@
     <h2 class="entry-title"><a href="{{ get_permalink() }}">{!! get_the_title() !!}</a></h2>
     @include('partials/entry-meta')
   </header>
-  <div class="entry-summary">
+  <main class="entry-summary">
```

Comment:
This (and the one in `partials.content-single`) should remain as `<div>` since the `layouts.app` already wraps the content in `<main>`.

> A document must not have more than one main element that does not have the hidden attribute specified.

https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element

Reply:
Will replace this; thank you for the working group reference. I've been operating under the outdated understanding that nested `<main>`s were fine (if within a semantic element like `<article>`), and that the one-per rule applied to root. But, this has clearly not been the case for like two years.

Reply:
Addressed here: https://github.com/roots/sage/pull/2144/commits/7bf9810b3b7414c85d803becbc23b18e8895f89b